ldk/go/examples/storage/Makefile

```diff
@@ -0,0 +1,48 @@
+REPO := github.com/open-olive/loop-development-kit/ldk/go/examples/storage
+PKG := examplego
+
+GIT_COMMIT := $(shell git rev-list -1 HEAD)
+GIT_BRANCH := $(if $(shell echo $$BRANCH_NAME),$(shell echo $$BRANCH_NAME),$(shell git rev-parse --abbrev-ref HEAD))
+VERSION := $(if $(shell echo $$VERSION),$(shell echo $$VERSION),"0.0.0-localdev-unset")
+GOPATH := $(shell go env GOPATH)
+
+install:
+# The following command is needed to allow go get to clone from private bitbucket repos
+	@ git config --global url."git@bitbucket.org:".insteadOf "https://bitbucket.org/"
+	go mod download
+	go mod tidy
+
+build/darwin-amd64/plugin:
+	go build \
+	     -ldflags "-X ${REPO}/controller/controller.GitBranch=${GIT_BRANCH} -X ${REPO}/sensor/sensor.GitCommit=${GIT_COMMIT} -X ${REPO}/sensor/sensor.Version=${VERSION}" \
```

Comment:
This line should probably change to:
`-ldflags "-X ${REPO}/loop.GitBranch=${GIT_BRANCH} -X ${REPO}/loop.GitCommit=${GIT_COMMIT} -X ${REPO}/loop.Version=${VERSION}" \`

Reply:
Thanks for pointing this out. I went ahead and made you suggested changes.
These flags don't do anything right now because those variables are not defined in the package. We have a card in the backlog to make these do something.